server/index.js

```diff
@@ -141,6 +141,19 @@ app.prepare().then(() => {
       html,
     };
 
+    const dublicateMailOptions = {
+      from: 'no-reply@keenethics.com',
+      to: 'oleh.romanyuk@keenethics.com',
```

Comment:
look here https://nodemailer.com/about/#example you can have multi `to`

Reply:
Thank you. Fixed.